scripts/SMD_chip_package_rlc-etc/size_definitions/size_resistor_chip_wide_body.yaml

```diff
@@ -13,7 +13,7 @@ SMD_0612:
     ipc_reference: "ipc_spec_larger_or_equal_0603"
     size_info: '(Body size source: https://www.vishay.com/docs/20019/rcwe.pdf)'
 
-SMD_0815:
+R_0815:
```

Comment:
The datasheet for that no longer exists.

I suggest to use this one: https://www.bourns.com/docs/Product-Datasheets/CRK.pdf
![image](https://user-images.githubusercontent.com/17516572/93024876-d3f60900-f5f9-11ea-8818-544a844c9c5c.png)

Reply:
OK. Updated. Note that there are dimensional changes here, especially a much tighter terminal width dimension so the resulting footprint will change. Again, shall I evaluate before doing this or would you like to?

Also, please take a look at my question at https://github.com/pointhi/kicad-footprint-generator/pull/438#issuecomment-538133156. Should those all be merged into one YAML file? I'm not sure why wide body needs a separate YAML file, but if there's an advantage to breaking them up I have no problem leaving it alone.

Reply:
As I did above, here is the current footprint:
- Pad centers: +/-0.85mm
- Pad width: 1.35mm
- Pad inside: 0.35mm
- Pad outside: 3.05mm

And with the dimensions from above:
- Pad centers: +/-1.0375mm
- Pad width: 1.125mm
- Pad inside: 0.95mm
- Pad outside: 3.2mm

That is a huge increase in pad minimum distance! I'm thinking perhaps a bit more surveying of the parts in the market to ascertain more of an average might be in order. You?

Reply:
Here is a comparison of the recommended burns landing pattern (top) with the current master footprint (bottom). 
![image](https://user-images.githubusercontent.com/17516572/93252460-d4c3a200-f795-11ea-90e6-829f627c84ad.png)

The only other manufacturer on digikey I could find was this - which is identical to the Burns part.
![image](https://user-images.githubusercontent.com/17516572/93253000-8e227780-f796-11ea-9215-09397a59a9c6.png)
https://www.susumu.co.jp/common/pdf/n_catalog_partition07_en.pdf

Asssuming the extreme values on tolerance (Burns) the resistor would be 3.45mm wide, with the terminal 0.7mm. So the inner-most-edge of metal is 3.45/2 - 0.7 = 1.025. 
So a pad-inside position of 0.95mm seems perfectly fine for me.

Reply:
The 3.75+/-0.3mm matches on Bourns and Susumu, but annoyingly the other dimensions is 2mm vs 2.3mm. I also found https://www.seielect.com/catalog/sei-csr_csrn.pdf which is 3.75x2mm, matching Susumu. And 80mil is 2.032mm, so 2mm nominal is much closer to matching the '08' in the package name than 2.3mm. Which dimensions should you like to go with?

Reply:
I think it is reasonable to go with 2.0mm nominal.
These resistors sit in lots of paste and are relatively heavy with lots of leverage, I don't expect them to tombstone even when the pad geometry is bad.

I suggest to add part of of this discussion as comment to the .yaml file. Reference both the sumusu and sei datasheets. That ensures users, ourselves, and future contributors can understand why that value is there.

Unfortunately the recommended landing pattern of sei is quite different.
I would suggest to let the generator do its job. Then we can inspect the resulting footprint and compare it with sei and burns recommended footprints.

Reply:
Using 2+/-0.2mm gives:
Pad centers: +/-0.9375mm
Pad width: 1.125mm
Pad inside: 0.75mm
Pad outside: 3mm

I think that should be fine based on the available 0815 resistors we found so I'm resolving this. I've added comments.